lib/matplotlib/tests/test_axes.py

```diff
@@ -5301,16 +5255,13 @@ def test_title_pad():
 
 def test_title_location_roundtrip():
     fig, ax = plt.subplots()
-    # set default title location
-    plt.rcParams['axes.titlelocation'] = 'center'
-
     ax.set_title('aardvark')
     ax.set_title('left', loc='left')
     ax.set_title('right', loc='right')
 
     assert 'left' == ax.get_title(loc='left')
     assert 'right' == ax.get_title(loc='right')
-    assert 'aardvark' == ax.get_title(loc='center')
```

Comment:
Probably should keep both versions of this.

Reply:
oops, that was removed by mistake.  restored.